var/spack/repos/builtin/packages/mpitrampoline/package.py

```diff
@@ -37,6 +51,31 @@ class Mpitrampoline(CMakePackage):
     def cmake_args(self):
         return [self.define_from_variant('BUILD_SHARED_LIBS', 'shared')]
 
+    @property
+    def headers(self):
+        hdrs = HeaderList(find(self.prefix, 'mpi.h', recursive=True))
```

Comment:
It would be better if we point to a subfolder and we don't recurse. Is `mpitrampoline` always installing headers with the same layout?

Reply:
Done.